Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles.Tests/Lib/Helpers.cs

```diff
@@ -169,5 +140,115 @@ internal static void SetTestFileLastModified(string path, DateTime date)
             client.Disconnect();
         }
     }
+
+    internal static string GetServerFingerprintAsMD5HexString()
+    {
+        var fingerprint = "";
+        using (var client = new SftpClient(_dockerAddress, 2222, _dockerUsername, _dockerPassword))
+        {
+            client.HostKeyReceived += delegate (object sender, HostKeyEventArgs e)
+            {
+                fingerprint = BitConverter.ToString(e.FingerPrint).Replace("-", ":");
+                e.CanTrust = true;
+            };
+            client.Connect();
```

Comment:
Why do we need this here? Aren't we anyway connecting at some point? For downloading something in a loop we are introducing quite an overhead (2x the connections).

Reply:
The connection is needed to get to the point when the event client.HostKeyReceived is made. This is the only way to acquire the real server fingerprint for the tests. However I can fetch the server fingerprint in one helper method and store them to global variables.

Reply:
Changes the implementation to fetch the server fingerprint and host key as byte arrays in OneTimeSetup. Modification is done later to them.